cmake/modules/StandaloneOverlay.cmake

```diff
@@ -10,9 +10,13 @@ endif()
 
 set(CMAKE_INSTALL_PREFIX "${SWIFT_DEST_ROOT}${TOOLCHAIN_DIR}/usr")
 
+set(SWIFT_STDLIB_BUILD_TYPE "Release" CACHE STRING
+    "Build type for the Swift standard library and SDK overlays [Debug, RelWithDebInfo, Release, MinSizeRel]")
+set_property(CACHE SWIFT_STDLIB_BUILD_TYPE PROPERTY
```

Comment:
Why the property?  The `set(...CACHE...)` should be sufficient.

Reply:
I'm just duplicating what's already in the top-level `CMakeLists.txt` file here.